AnnoyancesFilter/Popups/sections/subscriptions_specific.txt

```diff
@@ -48,6 +48,7 @@ jiji.com,edp24.co.uk,technologyreview.com,inews.co.uk,nordbayern.de,thelocal.ch,
 !
 ! SECTION: Subscriptions - Regular rules
 !
+allrecipes.com,foodandwine.com,health.com,investopedia.com,lifewire.com,mydomaine.com,parents.com,simplyrecipes.com,thespruce.com,thespruceeats.com,thesprucepets.com,travelandleisure.com,treehugger.com,tripsavvy.com,verywellfamily.com,verywellfit.com,verywellhealth.com,verywellmind.com$$div[data-a11y-dialog="newsletter-dialog"]
```

Comment:
The rule shoud be changed, but tell me have you seen an issue with page scrolling?
I've seen it here `foodandwine.com`. What about other sites? It should be the same, but please check it.

Reply:
It may be, so that deleting an element before parsing the page automatically interrupts JavaScript to block scrolling.

Reply:
Seems these rules work fine on my end.

```adblock
allrecipes.com,foodandwine.com,health.com,investopedia.com,lifewire.com,mydomaine.com,parents.com,simplyrecipes.com,thespruce.com,thespruceeats.com,thesprucepets.com,travelandleisure.com,treehugger.com,tripsavvy.com,verywellfamily.com,verywellfit.com,verywellhealth.com,verywellmind.com#$#div[data-a11y-dialog="newsletter-dialog"] { display: none !important; }
allrecipes.com,foodandwine.com,health.com,investopedia.com,lifewire.com,mydomaine.com,parents.com,simplyrecipes.com,thespruce.com,thespruceeats.com,thesprucepets.com,travelandleisure.com,treehugger.com,tripsavvy.com,verywellfamily.com,verywellfit.com,verywellhealth.com,verywellmind.com#$#html { overflow: auto !important; }
```

Reply:
Could be added for Chromium/Safari WebExtension if HTML filtering works fine for Apps and Firefox.

Reply:
It can be added this way. This type of the rules are supported by our products.
@wavygecko can you do the changes?

Reply:
I've done it here.